src/index.js

```diff
@@ -1,3 +1,148 @@
-// eslint-disable-next-line no-unused-vars
-import _ from 'lodash';
 import './style.css';
+
+let gameId = ''; // Store the ID of the game created by you.
+
+// Function to display an error message in the DOM.
+const displayErrorMessage = (message) => {
+  const errorContainer = document.querySelector('.error-container');
+  errorContainer.textContent = message;
+};
+
+// Function to create a new game and save gameId to local storage.
+const createGame = async () => {
+  try {
+    const response = await fetch(
+      'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/',
+      {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({ name: 'Barcelona' }),
+      },
+    );
+    const data = await response.json();
+    const gameId = data.result.split(':')[1].trim();
```

Comment:
You've done a good job with the `createGame()` method, which successfully creates a new game and stores the game ID in local storage. However, there is a small issue to address. It appears that you may have forgotten to remove the unwanted part of the text after the game ID when storing it in local storage.

To correct this, please update your code to save only the proper game ID, without any additional text or characters. Here's the desired format:

| Your                       | Desired                  |
| -------------------------- | ------------------------- |
| `1f2wq4HohBvkss7Uihuk added.` | `1f2wq4HohBvkss7Uihuk`   |
| ![image](https://github.com/efmomoh/Leaderboard/assets/55840999/23c71690-de33-4a9a-86bf-53c96e8222ed) | ![image](https://github.com/efmomoh/Leaderboard/assets/55840999/24d226da-b351-4004-9f58-fe440ff10c6d)|


By making this adjustment, you'll ensure that the game ID is stored correctly in local storage, without any unnecessary information. Keep up the great work! 🌟🎮

Reply:
Thank you @shahadat3669  for your thorough feedback given. I will implement them, thank you again 👍